app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -44,6 +51,10 @@ class HomeFragmentPresenter @Inject constructor(
   private lateinit var allTopicsViewModel: AllTopicsViewModel
   private lateinit var topicListAdapter: TopicListAdapter
   private lateinit var binding: HomeFragmentBinding
+  private var profileIdFromIntent: Int = -1
```

Comment:
Lets make this convention clear to avoid any confusion.

If the id is `int` we can name that as `internalProfileId`
and if it is of type `ProfileId` we can directly name that as `profileId`.

Reply:
Updated to internalProfileId